website/docs/guides/version-4-upgrade.html.md

```diff
@@ -200,4 +209,21 @@ resource "aws_batch_compute_environment" "test" {
 
 Default values
 
-
+## EC2-Classic Resource and Data Source Support
```

Comment:
Not entirely sure if this should live in this guide or the link above in the note should suffice 🤔